workbench-core/authorization/src/dynamicAuthorization/dynamicPermissionsPluginInputs.ts

```diff
@@ -0,0 +1,318 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Action } from '../action';
+import { Effect } from '../permission';
+
+/**
+ * Request object for DynamicPermissionsPlugin's createGroup
+ */
+export interface CreateGroupRequest {
```

Comment:
These interfaces should live in their own file to help prevent merge conflicts when two people have to make simultaneous changes to this file.